rdr_service/services/google_sheets_client.py

```diff
@@ -44,6 +41,24 @@ def __init__(self, spreadsheet_id, service_key_id, tab_offsets=None):
             'offset_str': offset
         } for tab_name, offset in tab_offsets.items()} if tab_offsets else {}
 
+    def _build_service(self):
+        # The Google API client uses sockets, and the requests can take longer than the default timeout.
+        # The proposed solution is to increase the default timeout manually
+        # https://github.com/googleapis/google-api-python-client/issues/632
+        # The socket seems to be created when calling discover.build, so this temporarily increases the timout for
```

Comment:
:nit timeout

Reply:
Thanks for catching that. Apparently it's not the first time I've done that 😂